lib/ansible/modules/monitoring/circonus_annotation.py

```diff
@@ -151,12 +143,15 @@ def main():
             api_key=dict(required=True, no_log=True)
             )
         )
+
+    if not HAS_REQUESTS:
+        module.fail_json(msg='requests is required for this module')
```

Comment:
I think @bcoca was suggesting that there should be a version check in the code.  Something like:
``` python
from distutils.version import LooseVersion
from ansible.module_utils.six import PY3

[...]

def check_requests_dep(module):
    if not HAS_REQUESTS:
        module.fail_json(msg='requests (version 1.0+ for Python2 or 2.0+ for Python3) is required for this module')

    if PY3:
        if LooseVersion(requests.__version__) < LooseVersion('2.0'):
            module.fail_json(msg='On Python3, this module requires requests-2.0 or higher')
    else:
        if LooseVersion(requests.__version__) < LooseVersion('1.0'):
            module.fail_json(msg='On Python2, this module requires requests-1.0 or higher')

[....]

# in def main():
check_requests_dep(module)
```